Editor/UI/ProjectAuditorWindow.cs

```diff
@@ -419,22 +437,252 @@ private void DrawCallHierarchy(CallTreeNode callTree)
+            {
+                int selectionCount = selectionDict[name];
+                if (selectionCount <= 0)
+                    continue;
+                int itemCount = dict[name];
+                if (itemCount == 1)
+                    individualItems.Add(name);
+                else if (selectionCount != itemCount)
+                    individualItems.Add(string.Format("{0} ({1} of {2})", name, selectionCount, itemCount));
+                else
+                    individualItems.Add(string.Format("{0} (All)", name));
+            }
+
+            // Maintain alphabetical order
+            individualItems.Sort(CompareUINames);
+
+            if (individualItems.Count == 0)
+                return "None";
+
+            string selectedText = string.Join(", ", individualItems.ToArray());
+            return selectedText;
+        }
+        
+        private int CompareUINames(string a, string b)
+        {
+            string[] aTokens = a.Split(':');
+            string[] bTokens = b.Split(':');
+
+            if (aTokens.Length > 1 && bTokens.Length > 1)
+            {
+                var aThreadName = aTokens[0].Trim();
+                var bThreadName = bTokens[0].Trim();
+
+                if (aThreadName == bThreadName)
+                {
+                    string aThreadIndex = aTokens[1].Trim();
+                    string bThreadIndex = bTokens[1].Trim();
+
+                    if (aThreadIndex == "All" && bThreadIndex != "All")
+                        return -1;
+                    if (aThreadIndex != "All" && bThreadIndex == "All")
+                        return 1;
+
+                    int aGroupIndex;
+                    if (int.TryParse(aThreadIndex, out aGroupIndex))
+                    {
+                        int bGroupIndex;
+                        if (int.TryParse(bThreadIndex, out bGroupIndex))
+                        {
+                            return aGroupIndex.CompareTo(bGroupIndex);
+                        }
+                    }
+                }
+            }
+
+            return a.CompareTo(b);
+        }
+        
+        private void DrawSelectedText(string text)
+        {
+#if UNITY_2019_1_OR_NEWER
+            GUIStyle treeViewSelectionStyle = "TV Selection";
+            GUIStyle backgroundStyle = new GUIStyle(treeViewSelectionStyle);
+
+            GUIStyle treeViewLineStyle = "TV Line";
+            GUIStyle textStyle = new GUIStyle(treeViewLineStyle);
+#else
+            GUIStyle textStyle = GUI.skin.label;
+#endif
+
+            GUIContent content = new GUIContent(text, text);
+            Vector2 size = textStyle.CalcSize(content);
+            Rect rect = EditorGUILayout.GetControlRect(GUILayout.MaxWidth(size.x), GUILayout.Height(size.y));
+            if (Event.current.type == EventType.Repaint)
+            {
+#if UNITY_2019_1_OR_NEWER
+                backgroundStyle.Draw(rect, false, false, true, true);
+#endif
+                GUI.Label(rect, content, textStyle);
```

Comment:
I noticed that if you select several assemblies the string is cropped. Perhaps we should display the first two or so selected assembly names only then "Assembly-CSharp, Unity.TextMeshPro..." or we should fix the parent UI element to expand horizontally.

Reply:
Fixed the parent UI element to fill the width of the window, allowing the maximum possible space for the summary string

Reply:
Looks much better now!